tracer/test/Datadog.Trace.Tests/Telemetry/config_prefix_block_list.json

```diff
@@ -118,6 +118,8 @@
   "DD_Ssrf_",
   "DD_TRACE_Ldap_",
   "DD_Ldap_",
+  "DD_TRACE_HardcodedSecret_",
+  "DD_HardcodedSecret_",
```

Comment:
If you keep the `IntegrationId` don't forget to create a PR in Datadog/dd-go for thsese changes 🙂